src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1633,6 +1646,11 @@ public override BoundNode VisitAssignmentOperator(BoundAssignmentOperator node)
             VisitLvalue(node.Left);
             VisitRvalue(node.Right);
 
+            if (node.RefKind != RefKind.None)
+            {
+                WriteArgument(node.Right, node.RefKind, method: null);
```

Comment:
>WriteArgument(node.Right, node.RefKind, method: null); [](start = 16, length = 54)

It looks like there is no test that tests effect of this line. Please add one if that is the case.